metadata.yaml

```diff
@@ -2,6 +2,18 @@ name: content-cache-k8s
 display-name: Content-cache
 description: |
   A service providing local caches of HTTP servers and useful for building content delivery networks (CDN).
+
```

Comment:
Start with `A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
  deploying and managing`